Library/Formula/node.rb

```diff
@@ -1,130 +1,23 @@
-# Note that x.even are stable releases, x.odd are devel releases
 class Node < Formula
   desc "Platform built on Chrome's JavaScript runtime to build network applications"
   homepage "https://nodejs.org/"
-  url "https://nodejs.org/dist/v0.12.7/node-v0.12.7.tar.gz"
-  sha256 "b23d64df051c9c969b0c583f802d5d71de342e53067127a5061415be7e12f39d"
-  head "https://github.com/nodejs/node.git", :branch => "v0.12"
-  revision 1
-
-  bottle do
-    sha256 "15689cc474a79975eaa6d791b24e6fa021494839c9b691ac307d74acefc5f834" => :yosemite
-    sha256 "a7a7d37c6e5088ed3f58b867d4d246851715d3a4f2f3b4b3c40cc7452ff6728c" => :mavericks
-    sha256 "374f3c5b576e4173590b8413e9941df121a84f46bd48161fc758e1f7d42e0402" => :mountain_lion
-  end
+  url "https://nodejs.org/dist/v4.0.0/node-v4.0.0.tar.gz"
+  sha256 "e110e5a066f3a6fe565ede7dd66f3727384b9b5c5fbf46f8db723d726e2f5900"
 
   option "with-debug", "Build with debugger hooks"
-  option "without-npm", "npm will not be installed"
-  option "without-completion", "npm bash completion will not be installed"
+  option "with-icu4c", "Build with Intl (icu4c) support"
 
-  deprecated_option "enable-debug" => "with-debug"
-
-  depends_on :python => :build if MacOS.version <= :snow_leopard
   depends_on "pkg-config" => :build
-  depends_on "openssl" => :optional
-
-  # https://github.com/nodejs/node-v0.x-archive/issues/7919
-  # https://github.com/Homebrew/homebrew/issues/36681
   depends_on "icu4c" => :optional
```

Comment:
I.e. (in re: https://github.com/Homebrew/homebrew/pull/43714#discussion_r39027144) one would remove this line as well, so the homebrew icu4c formula doesn't override the system ICU lib.